.github/workflows/validate-bicep.yaml

```diff
@@ -33,17 +33,15 @@ jobs:
     name: Validate Bicep Code
     runs-on: ubuntu-latest
     steps:
+      - name: Setup and verify bicep CLI
```

Comment:
there seems to be some lag between the bicep download and the verify step so moving this up to mitigate that